src/styles/components/helptour.scss

```diff
@@ -0,0 +1,175 @@
+/*
+ * Spreed WebRTC.
+ * Copyright (C) 2013-2014 struktur AG
+ *
+ * This file is part of Spreed WebRTC.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+$darken-tour-opacity: .4;
+
+#helptour {
+}
+
+.helptour {
+
+  .tourControl {
+    text-align: center;
+
+    hr {
+        margin: 10px 0;
+    }
+  }
+
+  .in {
+    display: block;
+  }
+
+  .slideInfo {
+    float: right;
+  }
+
+  .popover {
+    opacity: 0.95;
+  }
+
+  .roomPane {
+    left: 10px;
+    top: 97px;
+
+    .arrow {
+      left: 10%;
+    }
+  }
+
+  .buddyListPane {
+    left: initial;
+    right: 280px;
+    top: 70px;
+    .arrow {
+      top: 20%;
+    }
+  }
+
+  .chatPane {
+    left: initial;
+    right: 540px;
+    top: 70px;
+    .arrow {
+      top: 20%;
+    }
+  }
+
+  .settingsPane {
+    left: initial;
+    right: 540px;
+    top: 200px;
+    .arrow {
+      top: 20%;
+    }
+  }
+
+  .optionsPane {
+    float: right;
+    margin-right: 20px;
+    position: relative;
+    top: 71px;
+  }
+
+  .welcomePane {
+    position: relative;
+    margin: 0 auto;
+    top: 150px;
+
+    .btn {
+      margin-top: 10px;
+    }
+  }
+}
+
+.arrow+.secondArrow {
+  border-width: 11px;
+  &:after {
+    border-width: 10px;
+    content: "";
+  }
+}
+
+.arrow+.secondArrow,
+.arrow+.secondArrow:after {
+  border-color: rgba(0, 0, 0, 0);
+  border-style: solid;
+  display: block;
+  height: 0;
+  position: absolute;
+  width: 0;
+}
+
+.secondArrow.right {
+  border-left-width: 0;
+  border-right-color: #999;
+  border-right-color: rgba(0, 0, 0, 0.25);
+  left: -11px;
+  margin-top: -11px;
+  top: 50%;
+
+  &:after {
+    border-left-width: 0;
+    border-right-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`